docs/guides/cache/azblob.md

```diff
@@ -2,61 +2,57 @@
 
 > **Warning**
 >
-> The `azblob` cache is currently unreleased. You can use it today, by using
-> the `moby/buildkit:master` image in your Buildx driver.
+> This cache backend is unreleased. You can use it today, by using the
+> `moby/buildkit:master` image in your Buildx driver.
+{: .warning }
 
 The `azblob` cache store uploads your resulting build cache to
 [Azure's blob storage service](https://azure.microsoft.com/en-us/services/storage/blobs/).
 
 > **Note**
 >
-> The `azblob` cache storage backend requires using a different driver than
-> the default `docker` driver - see more information on selecting a driver
-> [here](../drivers/index.md). To create a new docker-container driver (which
-> can act as a simple drop-in replacement):
+> This cache storage backend requires using a different driver than the default
+> `docker` driver - see more information on selecting a driver
+> [here](../drivers/index.md). To create a new driver (which can act as a simple
+> drop-in replacement):
 >
 > ```console
 > docker buildx create --use --driver=docker-container
 > ```
 
-To import and export your cache using the `azblob` storage backend we use the
-`--cache-to` and `--cache-from` flags and point it to our desired blob using
-the required `account_url` and `name` parameters:
+## Synopsis
 
 ```console
-$ docker buildx build --push -t <user>/<image> \
-  --cache-to type=azblob,account_url=https://myaccount.blob.core.windows.net,name=my_image \
-  --cache-from type=azblob,account_url=https://myaccount.blob.core.windows.net,name=my_image
+$ docker buildx build . --push -t <registry>/<image> \
+  --cache-to type=azblob,name=<cache-image>[,parameters...] \
+  --cache-from type=azblob,name=<cache-image>[,parameters...]
 ```
 
-## Authentication
-
-To authenticate to Azure to read from and write to the cache, the following
-parameters are required:
+`--cache-to` options:
 
-* `secret_access_key`: secret access key
-  * specifies the primary or secondary account key for your Azure Blob
-    Storage account. [Azure Blob Storage account keys](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage)
+- `account_url`: the base address of the blob storage account, for example:
+  `https://myaccount.blob.core.windows.net`
+- `name`: the name of the cache image.
+- `secret_access_key`: specifies the
```

Comment:
*sorry*, super minor nitpick :smile: I think if the parameters are referenced in the authentication section later on, we should add a little note after them to point users in the right direction.

Also, I think it would look nice if we grouped shared parameters at the top of both lists? Not sure on that though.

I think `account_url` and `secret_access_key` are also `--cache-from` parameters as well here.

Reply:
hey, nice idea. I tried adding a common section for parameters. whaddya think?

Reply:
I like the common parameters, it reads really well :+1: